### ISSUE TITLE
IconOrientation.Top don't work in button from NuGet

### ISSUE THREAD

**Issue by Symorp** [robinmanuelthiel/flexbutton#86]
Hi first of all, thank you for your plugin and nice job.   

**When I try to use your plugin downloaded from NuGet IconOrientation.Top doesn't work, the icon is still on left side**
*But when I tried download repo from Github and add it to my project it works.*

Code (simply):
```
<flex:FlexButton
                Grid.Row="1"
                Grid.Column="0"
                Padding="0"
                BackgroundColor="Transparent"
                Clicked="Clicked"
                ForegroundColor="#444444"
                HasShadow="false"
                HighlightBackgroundColor="Transparent"
                HighlightBorderColor="Transparent"
                HighlightForegroundColor="#707070"
                HorizontalOptions="CenterAndExpand"
                Icon="icon"
                IconOrientation="Top"
                VerticalOptions="CenterAndExpand" />
````
**To Reproduce**
*Steps to reproduce the behavior:*

**Expected behavior**
*Icon above text*

**Please complete the following information:**
- Which version of the FlexButton do you use?
0.11.0
- Which version of Xamarin.Forms do you use?
4.4.0.991265
- Which OS are you talking about?
Android and iOS

**Comment by fgbustoa** [comment]
Confirmed

**Comment by robinmanuelthiel** [comment]
Fixed in version 0.12.0
